### ISSUE TITLE
Platform not found in 0.95

### ISSUE THREAD

**Issue by to4ko** [custom-components/sensor.unifigateway#10]
Updating HA to 0.95 got platform not found error during configuration check. Files placed in custom_components\unifigateway was working fine in previous version HA

**Comment by kaosmagix** [comment]
I have the same issue. Effect is that template sensors based on this are no longer populating (and graphs in lovelace no longer display values)

**Comment by nenadmilano** [comment]
Same issue here, seems like a breaking change in 0.95 affects this platform: 

- Replace pyunifi with aiounifi in UniFi device tracker
- UniFi simplify update

https://www.home-assistant.io/blog/2019/06/26/release-95/

**Comment by MTrab** [comment]
+1 on this

**Comment by goncalvespedro** [comment]
+1 on this too

**Comment by goncalvespedro** [comment]
Strange I must have rebooted a lot and removed some cached info and it's now working, not too sure what happened maybe pyunifi is still there :)

**Comment by KBrygger** [comment]
I also face this issue right after upgrading to 0.95. RPi rebooted several times and still same error when checking configuration:

Platform not found: sensor.unifigateway

**Comment by goncalvespedro** [comment]
I also have the integration of unifi installed, try this if you have hassio

Regards
Pedro

On Fri, 28 Jun 2019, 19:40 KBrygger, <notifications@github.com<mailto:notifications@github.com>> wrote:

I also face this issue right after upgrading to 0.95. RPi rebooted several times and still same error when checking configuration:

Platform not found: sensor.unifigateway

**Comment by sergiorademacher** [comment]
+1.  Platform not found after upgrade to 0.95

**Comment by KBrygger** [comment]
@goncalvespedro: How do one remove cached info?

**Comment by goncalvespedro** [comment]
I'm not online to give you exact location, but under the unifigateway folder you should only have 3 files, there is an extra folder in there that starts with "_" delete that and try reboot again.

Regards
Pedro

On Fri, 28 Jun 2019, 20:00 KBrygger, <notifications@github.com<mailto:notifications@github.com>> wrote:

@goncalvespedro<https://github.com/goncalvespedro>: How do one remove cached info?

**Comment by goncalvespedro** [comment]
Mine has stopped working again 😭

On Fri, 28 Jun 2019, 20:24 Pedro Gonçalves, <pedro.goncalves@live.com<mailto:pedro.goncalves@live.com>> wrote:
I'm not online to give you exact location, but under the unifigateway folder you should only have 3 files, there is an extra folder in there that starts with "_" delete that and try reboot again.

Regards
Pedro

On Fri, 28 Jun 2019, 20:00 KBrygger, <notifications@github.com<mailto:notifications@github.com>> wrote:

@goncalvespedro<https://github.com/goncalvespedro>: How do one remove cached info?

**Comment by goncalvespedro** [comment]
Okay I powered down the VM and started it up again and it's all working again. Apologies for all the messages.

On Fri, 28 Jun 2019, 22:05 Pedro Gonçalves, <pedro.goncalves@live.com<mailto:pedro.goncalves@live.com>> wrote:
Mine has stopped working again 😭

On Fri, 28 Jun 2019, 20:24 Pedro Gonçalves, <pedro.goncalves@live.com<mailto:pedro.goncalves@live.com>> wrote:
I'm not online to give you exact location, but under the unifigateway folder you should only have 3 files, there is an extra folder in there that starts with "_" delete that and try reboot again.

Regards
Pedro

On Fri, 28 Jun 2019, 20:00 KBrygger, <notifications@github.com<mailto:notifications@github.com>> wrote:

@goncalvespedro<https://github.com/goncalvespedro>: How do one remove cached info?

**Comment by cdrom1028** [comment]
@goncalvespedro are you saying the deleting the __pycache__ directory and rebooting solves the issue of Platform not found after upgrade to 0.95?

**Comment by nenadmilano** [comment]
> @goncalvespedro are you saying the deleting the **pycache** directory and rebooting solves the issue of Platform not found after upgrade to 0.95?

Did not work for me at least. Still getting the Error `Platform not found: sensor.unifigateway`

**Comment by jjanderson** [comment]
I have the same issue... deleted the pycache directory and rebooted, When I run "check config" I get:

```
Configuration invalid
Platform not found: sensor.unifigateway
```

**Comment by goncalvespedro** [comment]
Try updating the files in the unifigateway folder from GitHub, here are mine attached.

**Comment by nenadmilano** [comment]
Thanks @goncalvespedro but i installed the plugin via HACS and therefore i have the latest version from the git repository. Meanwhile i reinstalled it manually and via HACS but still the same error.

**Comment by kaosmagix** [comment]
Same for me. Removing the cache and manually installing the files again doesn't help. With the sensor active in the files HA no longer validates and doesn't restart. Sensors don't receive any values

**Comment by jjanderson** [comment]
I downloaded a fresh set of files, removed the cache directory, rebooted and still have the same result:

```
Configuration invalid

Platform not found: sensor.unifigateway
```

**Comment by jnalley20** [comment]
Same issue, but first time using this custom component through HACS.

**Comment by goncalvespedro** [comment]
Hi try rebooting the whole system not just Home Assistant, mine is hit and miss to get it working again.

**Comment by jnalley20** [comment]
sudo reboot
Issue still remains

**Comment by cdrom1028** [comment]
This is from the Developer ( @jchasey ) via the UniFi Security Gateway thread in Home Assistant Community Forum..

[https://community.home-assistant.io/t/unifi-security-gateway/71505/52](https://community.home-assistant.io/t/unifi-security-gateway/71505/52)

**Comment by jchasey** [comment]
There are similar problems being reported with the official Unifi device tracker component, which may be connected:

https://github.com/home-assistant/home-assistant/issues/24895

**Comment by Therkim** [comment]
I have manually resolved this issue on my clean install of 0.95.4 by activating my venv and installing pyunifi.

Best of luck.

**Comment by jchasey** [comment]
Anyone NOT using Hassio that is having the issue? Suspecting the dependency on pyunifi is the cause, ie this is not being installed as required.

**Comment by Therkim** [comment]
I am using the python venv manual install and was having the issue, resolved as above with manual pyunifi install.

**Comment by tmchow** [comment]
What are the disadvantages of manually installing pyunifi?

**Comment by MTrab** [comment]
It works again for me, with the latest release.

**Comment by jchasey** [comment]
Fixed in 0.2.3
